### ISSUE TITLE
Help! gitlab CI env detection?

### ISSUE THREAD

**Issue by sanyatuning** [mc1arke/sonarqube-community-branch-plugin#459]
**Describe the bug**
If I run sonar-scanner from gitlab-ci it forces branch analysis and ignores provided pull request parameters
If I run the same command on my machine it runs pull request analysis.

the command
```
~/sonar-scanner-3.1.0.1141-linux/bin/sonar-scanner -Dsonar.pullrequest.key=12 -Dsonar.pullrequest.branch=tp122317_fix_available_proj_only -Dsonar.pullrequest.base=master -Dsonar.ci.autoconfig.disabled=true -Dsonar.projectName=vcc-next/xas-proxy -Dsonar.projectKey=vcc-next-service-salesforce-xas-proxy -Dsonar.pullrequest.gitlab.repositorySlug=vcc-next/service/salesforce/xas-proxy
```

**Expected behavior**

expected output (on my machine)
```
INFO: Load active rules
INFO: Load active rules (done) | time=6373ms
INFO: Pull request 12 for merge into master from tp122317_fix_available_proj_only
INFO: SCM collecting changed files in the branch
INFO: Indexing files...
```

actual output (gitlab-ci)
```
INFO: Load active rules
INFO: Load active rules (done) | time=6407ms
INFO: Branch name: tp122317_fix_available_proj_only
INFO: Indexing files...
```

**Software Versions**
 - SonarQube Version: Community EditionVersion 8.9.2 (build 46101) - Docker version with plugin
 - Plugin Version: 1.8.1  (bundled)

**Additional context**
I guess some ENV variable was detected.
I already added "sonar.ci.autoconfig.disabled=true" but it didn't help.

**Comment by sanyatuning** [comment]
I found it, this helped me:
```bash
    unset CI_COMMIT_REF_NAME
```
